subsys/mpsl/pm/Kconfig

```diff
@@ -0,0 +1,13 @@
+#
```

Comment:
This file isnt sourced anywhere. Something like `rsource "pm/Kconfig"` should be added to `subsys/mpsl/Kconfig`.